code/modules/paperwork/pen.dm

```diff
@@ -332,3 +332,15 @@
 			P.contact_poison_poisoner = user.name
 			add_attack_logs(user, P, "Poison pen'ed")
 			to_chat(user, "<span class='warning'>You apply the poison to [P].</span>")
+
+// CHAMELEON PEN
+/obj/item/pen/chameleon
+	name = "chameleon pen"
+	desc = "It's a surprisingly boring-looking pen! Try using it to leave a signature."
```

Comment:
you want this pen to be identifiable by the name/descrition?

Reply:
I would love to hear more people thought about it. I think it might be hard to use pen that's why i have a hinting description for it.

Reply:
Apparently most people in chat i asked prefer it to not be so easily identifiable. So i will make it looking just like regular pen.